### DISCUSSION TITLE
Close the dialog, close the browser and ok()

### DISCUSSION THREAD

**Discussion by dansysanalyst** [magic-test/magic-test-laravel#26] [Ideas]
I was showing Magic Test to person who is not a power user but certainly knows his way on computers and he performed and assertion, closed the dialog and then closed the browser with the feeling "I am done here, terminal time".
When he entered `ok`:

This generates the error bellow:
  ```
  Facebook\WebDriver\Exception\NoSuchWindowException with message 'no such window: window was already closed
  (Session info: chrome=89.0.4389.82)'
  ```
  
Some possible solutions would be:

- Display a message when before the sell drops saying something like: "Tip: Do not close your browser before typing `finish()`".
- Would possible to display an alert like: " Attention: You have a Magic test in progress. First type `ok()` and then `finish()` on the console. Closing this window will may result in loss of information."

**Comment by mateusjatenee** [comment]
Hmmm yeah that's something we can't control. Maybe just adding this as a notice on the console as soon as they start Magic Test is best.

**Reply by dansysanalyst** [reply]
I made a few tests here but all solutions I found disturbed the testing flow.

I have added the notice on my messages PR.

Thanks
